src/controllers/dynakube/version/updater.go

```diff
@@ -164,13 +158,10 @@ func updateVersionStatusForTenantRegistry( //nolint:revive
 		"oldVersion", target.Version)
 
 	if taggedRef, ok := ref.(reference.NamedTagged); ok {
-		imageVersion, err := imageVersionFunc(ctx, imageUri, dockerCfg)
+		imageVersion, err := imageVersionFunc(ctx, &registry.Client{}, imageUri, dockerCfg)
```

Comment:
I would have a constructor for `&registry.Client{}`, just incase.
(nicer to look at and if the constructor's signature changes then IDEs will tell you)

Reply:
Fixed in [this commit](https://github.com/Dynatrace/dynatrace-operator/pull/1998/commits/c23e53e66bb19aaa680e8fd01e05fad805afd5a1)